support/scripts/mkefi.py

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env python3
+
+import argparse
+import subprocess
+import os
+import re
+from struct import pack
+
+ELF_MACHINE = {
+    'x86_64': 62,               # EM_X86_64
+    'arm64': 183                # EM_AARCH64
+}
+
+PE_MACHINE = {
+    'x86_64': 0x8664,           # IMAGE_FILE_MACHINE_AMD64
+    'arm64' : 0xaa64            # IMAGE_FILE_MACHINE_ARM64
+}
+
+PE_SEC_SZ = 40
+
+# COFF Characteristics
+COFF_HDR_CHRS = (
+                 0x0002     |  # IMAGE_FILE_EXECUTABLE_IMAGE
+                 0x0004     |  # IMAGE_FILE_LINE_NUMS_STRIPPED
+                 0x0200        # IMAGE_FILE_DEBUG_STRIPPED
+)
+
+# All sections that we obtain from the PT_LOAD Program Headers must be
+# writable, since we have to apply initial relocations. We are going to
+# change permissions ourselves afterwards anyway.
+PE_SEC_CHRS = (
+                0x00000040 |  # IMAGE_SCN_CNT_INITIALIZED_DATA
+                0x00d00000 |  # IMAGE_SCN_ALIGN_4096BYTES
+                0x20000000 |  # IMAGE_SCN_MEM_EXECUTE
+                0x40000000 |  # IMAGE_SCN_MEM_READ
+                0x80000000    # IMAGE_SCN_MEM_WRITE
+)
+
+MZ_PE_HDR = {
+    # MS-DOS Stub (Image Only)
```

Comment:
It would be better if the MZ header is defined in a separate dictionary, as that will make it easier to add support for the linux variant on arm64.